diff -aurN libgexiv2-0.4.1.orig/Makefile libgexiv2-0.4.1/Makefile
--- libgexiv2-0.4.1.orig/Makefile	2012-09-25 22:01:42.579933981 +0200
+++ libgexiv2-0.4.1/Makefile	2012-09-25 22:02:33.204153127 +0200
@@ -92,7 +92,7 @@
 DIST_TAR_GZ = $(DIST_TAR).gz
 
 EXT_PKGS_CFLAGS = `pkg-config --cflags $(EXT_PKGS)`
-EXT_PKGS_LDFLAGS = `pkg-config --libs $(EXT_PKGS)`
+EXT_PKGS_LDFLAGS = `pkg-config --libs $(EXT_PKGS)` -lm
 
 # REQUIRED_CFLAGS absolutely get appended to CFLAGS, whatever the
 # the value of CFLAGS in the environment
